playbooks/luna.yml

```diff
@@ -21,6 +21,6 @@
       - "--enable-foreman-proxy-plugin-ansible"
       - "--enable-foreman-proxy-plugin-discovery"
       - "--enable-foreman-proxy-plugin-openscap"
-      - "--enable-foreman-proxy-plugin-remote-execution-ssh"
+      - "{{ '--enable-foreman-proxy-plugin-remote-execution-script' if (foreman_repositories_version == 'nightly' or foreman_repositories_version is version('4.5', '>=')) else '--enable-foreman-proxy-plugin-remote-execution-ssh' }}"
```

Comment:
this playbook is only used for `centosX-luna` which isn't versioned and implies nightly.